app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -18,9 +18,8 @@ module.exports = React.createClass
     # console.log 'Old marks were', @state.oldSetOfMarks
     newSetOfMarks = []
     # Automatically select new marks.
-    annotations = nextProps.classification?.annotations ? []
-    annotation = annotations[annotations.length - 1]
-    if annotation?
+    annotation = nextProps.annotation
```

Comment:
Could you explain what this change does here? Looks like the previous code took the last annotation from the `classification.annotations` array. Is `nextProps.annotation` guaranteed to be that same annotation?